site/operational-guide/tasks.md

```diff
@@ -30,70 +42,76 @@ may not be successful, or indeed possible:
 
 However in some scenarios it may not be obvious that space has been
 lost, in which case you can check for it periodically with the
 following command on any peer:
 
     weave status ipam
 
-This will list the names of unreachable peers; if you are satisifed
+This command lists the names of unreachable peers. If you are satisifed
 that they are truly gone, rather than temporarily unreachable due to a
 partition, you can reclaim their space manually.
 
+###<a name="manually-reclaim-address-space"></a>Manually Reclaiming Address Space
+
 When a peer dies unexpectedly the remaining peers will consider its
 address space to be unavailable even after it has remained unreachable
-for prolonged periods; there is no universally applicable time limit
+for prolonged periods. There is no universally applicable time limit
 after which one of the remaining peers could decide unilaterally that
 it is safe to appropriate the space for itself, and so an
 administrative action is required to reclaim it.
 
-The `weave rmpeer` command is provided to perform this task, and must
-be executed on _one_ of the remaining peers. That peer will take
+The [`weave rmpeer`](https://www.weave.works/docs/net/latest/ipam/stop-remove-peers-ipam/)
+command is provided to perform this task, and must
+be executed on _one_ of the remaining peers. That peer will then take
 ownership of the freed address space.
 
-##<a name="cluster-upgrade"></a>Upgrade a Cluster
+##<a name="cluster-upgrade"></a>Upgrading a Cluster
 
 Protocol versioning and feature negotiation are employed in Weave Net
-to enable incremental rolling upgrades - each major release maintains
+to enable incremental rolling upgrades. Each major maintains
 the ability to speak to the preceding major release at a minimum, and
-connected peers only utilise features which both support. The general
-upgrade procedure is as follows:
+connected peers only utilize features which both support. 
 
-On each peer in turn:
+The general upgrade procedure is as follows:
 
-* Stop the old weave with `weave stop` (or `systemctl stop weave` if
+On each peer:
+
+* Stop the old Weave with `weave stop` (or `systemctl stop weave` if
   you're using a systemd unit file)
-* Download the new weave script and replace the existing one
-* Start the new weave with `weave launch <existing peer list>` (or
+* Download the new Weave script and replace the existing one
+* Start the new Weave with `weave launch <existing peer list>` (or
   `systemctl start weave` if you're using a systemd unit file)
 
-This will result in some downtime as the first launch with the new
-script has to pull the new container images; if you wish to minimise
-downtime you can download the new script to a temporary location
-first:
+To minimize downtime while the new script is pulling the new container images, 
+it is recommended that you download the new script to a temporary location first:
```

Comment:
The 2nd part is still redundant. Simply remove it, i.e. just say "To minimize downtime while the new script is pulling the new container images:"